03/05_array.php

```diff
@@ -0,0 +1,19 @@
+<?php 
+
+$life = [
+    'study' => [
+        '国語' => ['現代文', '古文', '漢文'],
+        '理科' => ['物理', '生物'],
+        '社会' => ['日本史', '現代社会']
+    ],
+    'club' => [
+        '運動部' => ['サッカー部', 'テニス部', 'バレー部', 'バスケ部'],
+        '文化部' => ['茶道部', '書道部', 'かるた部']
+    ]
+];
+
+$study = "{$life['study']['国語'][1]}、{$life['study']['理科'][0]}、{$life['study']['社会'][0]} ";
```

Comment:
```suggestion
$study = "{$life['study']['国語'][1]}、{$life['study']['理科'][0]}、{$life['study']['社会'][0]}";
```
最後の半角スペースは不要です。

$study、$clubの編集方法は、今の書き方でも問題ありません✨
ですが、要件の変更で結合する個数が増減する場合、これ以上長くなるとちょっと見づらいな。。とか、どこからどこまで消せばいいんだ。。となることが考えられます。
例えば以下の書き方だと、結合の増減にも容易に対応できそうな感じがしませんか？
```
$study =  $life['study']['国語'][1] . '、' .
          $life['study']['理科'][0] . '、' .
          $life['study']['社会'][0];
$club = $life['club']['運動部'][2] . 'と' .
        $life['club']['文化部'][0];
```
変数の編集方法にも色々あってどれが正解なのかは場面によって変わりますが、あんな方法こんな方法がこれからのカリキュラムでも出てきますので、選択肢を増やしていきましょう😄